### DISCUSSION TITLE
Can bao-hypervisor support macOS on Apple silicon?

### DISCUSSION THREAD

**Discussion by zlaazlaa** [bao-project/bao-hypervisor#175] [General]
I try to run bao-demos on macOS (Apple silicon) but failed. Is bao-hypervisor support this machine?
If not, what can I do to support it? I'd like to open a PR to solve it. The cross compiler now I use is [arm-gnu-toolchain-13.3.rel1-darwin-arm64-aarch64-none-elf.tar.xz](https://developer.arm.com/-/media/Files/downloads/gnu/13.3.rel1/binrel/arm-gnu-toolchain-13.3.rel1-darwin-arm64-aarch64-none-elf.tar.xz). It can not complete compile, maybe there will be other work to do. Can you give me some advice about it?

**Comment by josecm** [comment]
Hey @zlaazlaa. Could you please give us more details on what errors do you encounter? Is it the hypervisor itself, the guests (maybe linux/buildroot) that don't compile correctly or are there other issues maybe with the makefiles? Also, what platform are you targetting? QEMU?
